src/config/userSettings.js

```diff
@@ -19,6 +19,16 @@ const userSettings = {
     },
     {
       id: uuidv4(),
+      key: 'github',
```

Comment:
Please rename to 'github-pr' or something. Should be more specific in case we add other Github widgets later.